### ISSUE TITLE
Automated builds

### ISSUE THREAD

**Issue by cyberdrop-me** [wmw9/cyberdrop-dl#4]
I have never touched rust so maybe you could identify the issue.

I was trying to make an actions workflow to automatically generate builds for linux, windows, and mac to make it easier for people to get the tool up and running.  I was able to confirm that the generated windows build works but the linux build, and probably the mac build too, throws an error about tokio:

```
user@lxc /tmp/releasetmp # RUST_BACKTRACE=1 ./cyberdrop-dl https://cyberdrop.me/a/1
Albums to download: 1
     Overall [░░░░░░░░░░░░░░░░░░░░ 0/1]
thread 'tokio-runtime-worker' panicked at 'Error creating dir: Not a directory (os error 20)', /project/src/lib.rs:188:29
stack backtrace:
note: Some details are omitted, run with `RUST_BACKTRACE=full` for a verbose backtrace.
thread 'tokio-runtime-worker' panicked at 'called `Result::unwrap()` on an `Err` value: JoinError::Panic(...)', /project/src/lib.rs:65:17
stack backtrace:
Done!
```

The `/project/....` is from the github actions server, it kept that structure for some reason.

You can find the builds here https://github.com/cyberdrop-me/cyberdrop-dl/releases/tag/v0.3.1 and here's the workflow file https://github.com/cyberdrop-me/cyberdrop-dl/blob/main/.github/workflows/release.yml

I had to add a package to the Cargo.toml here https://github.com/cyberdrop-me/cyberdrop-dl/blob/main/Cargo.toml#L25 because the mac & linux build kept complaining about this error https://github.com/sfackler/rust-openssl/issues/1021

**Comment by evan-sm** [comment]
Thanks for contributing! I might get my first Pull Request soon 😇
Yeah, Rust often have issues compiling for different platforms. I'll look at it in my free time.

Also, I am thinking about to use ```rust-tls``` instead of ```openssl```, it works better when it comes to crossplatform compilation.

**Comment by evan-sm** [comment]
Ok, checked it! I think it should be **fixed** now. This bug was actually kinda funny when I discovered cause.

When you run ```./cyberdrop-dl``` binary in the same directory it tries to create folder named ```./cyberdrop-dl/``` to download album files, but since there's already binary file exists with same name ```./cyberdrop``` it panics haha. So I just changed default folder name for downloads to ```./cyberdrop.me/```

P.S: crates.io also updated
